README.md

```diff
@@ -173,7 +173,7 @@ element may be broken up into multiple chunks.
 This class reassembles these elements by buffering incomplete ones.
 
 The `Decoder` supports the same optional parameters as the underlying
-[`str_getcsv()`](http://php.net/str_getcsv) function.
+[`str_getcsv()`](https://php.net/str_getcsv) function.
```

Comment:
Should probably be replaced with canonical link, WDYT? 👍

```suggestion
[`str_getcsv()`](https://www.php.net/manual/en/function.str-getcsv.php) function.
```

Reply:
Oops, my fault to not look closer to the link. Thank you @clue for noticing this.